src/pages/Scholarships.jsx

```diff
@@ -43,6 +47,14 @@ function ScholarshipsPage() {
         (loading && <CircularProgress className={classes.progress} />) || (
           <ScholarshipList scholarships={scholarships} />
         )}
```

Comment:
Move this line below the `<Button>` you added. This'll make the button only show up when there are results